site/spi/Makefile-bits

```diff
@@ -16,26 +16,18 @@ ifeq ($(SP_OS), rhel7)
 
     ## If not overridden, here is our preferred LLVM installation
     ## (may be changed as new versions are rolled out to the facility)
-    LLVM_DIRECTORY ?= /shots/spi/home/lib/arnold/rhel7/llvm_4.0_final/
+    LLVM_DIRECTORY ?= /shots/spi/home/lib/arnold/rhel7/llvm_4.0
```

Comment:
Yeah, that's what I was thinking, but I forgot to do it. I'm out today, can you fix the link?